lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -192,6 +212,10 @@ function dosomething_northstar_transform_user($user, $password = null) {
 
   // Set values in ns_user if they are set.
   foreach ($optional as $ns_key => $drupal_key) {
+    if (! isset($user->$drupal_key)) {
```

Comment:
Can we combine this with the `isset` on [line 220](https://github.com/DoSomething/phoenix/pull/7023/files#diff-6fba28391e7b23bdda9b664d3a71c59fR220) in that case (by inlining the `$user->{$drupal_key}` part)?